trio/_core/_run.py

```diff
@@ -1150,7 +1151,7 @@ class GuestState:
     run_sync_soon_not_threadsafe = attr.ib()
     done_callback = attr.ib()
     unrolled_run_gen = attr.ib()
-    unrolled_run_next_send = attr.ib(factory=lambda: Value(None))
+    unrolled_run_next_send = attr.ib(factory=lambda: Value(None), type=object)
```

Comment:
I don't understand why I had to do this. 🤷

Reply:
I guess the type is actually `Outcome`?

Reply:
I get `trio/_core/_run.py:1154: error: Returning Any from function declared to return "object"` when trying anything else than object. I guess I need to understand how attrs typing support works.